cas-server-core-services/src/main/java/org/apereo/cas/services/ServiceRegistryInitializationConfiguration.java

```diff
@@ -0,0 +1,69 @@
+package org.apereo.cas.services;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.PostConstruct;
+
+
+/**
+ * Initializes Jpa service registry database with available JSON service definitions if necessary (based on configuration flag).
+ *
+ * @author Dmitriy Kopylenko
+ * @author Misagh Moayyed
+ * @since 5.0.0
+ */
+@Configuration("serviceRegistryInitializationConfiguration")
+public class ServiceRegistryInitializationConfiguration {
```

Comment:
And also, why don't we make this class a package-private? It's not really a "public API". And more general not - would be good to start making package-private other "internal" CAS components